content/projects/coursera/it-automation/course-1-python-crash-course/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+_db_id:
+available_flavours:
+- none
+content_type: project
+ready: true
+submission_type: link
+title: Coursera - IT Automation with Python - Course 1 - Crash Course on Python
+---
+
+Complete the above mentioned course on coursera
```

Comment:
add some submission instructions, i.e once you have completed this course take a screenshot of $whatever_you_need_to_see upload it and submit the link on the project card